Documentation/Examples/arangodump.json

```diff
@@ -64,6 +64,20 @@
     "section" : "",
     "type" : "boolean"
   },
+  "encryption.key-generator" : {
```

Comment:
@Simran-B , I am sure you will notice this, but just in case. We should keep in mind that we have added these two options already in the doc, manually, in PR https://github.com/arangodb/arangodb/pull/5881 . Are these changes added by Simon the result of the json file created automatically over enterprise binary or have them been added manually? if added manually we probably should consider to revert, as they will get lost and will duplicate text we added already in that PR.  If the generation of the json option file is now done using enterprise binary, we should take actions in that other PR anyway. Thanks,

Reply:
Yes I think I accidentally commiited these, I did not manually modify them